### DISCUSSION TITLE
Is there a BC object to specify a BC that depends on the time derivative of a variable?

### DISCUSSION THREAD

**Discussion by richmondodufisan** [idaholab/moose#28646] [Q&A General]
### Check these boxes if you have followed the posting rules.

- [X] Q&A General is the most appropriate section for my question
- [X] I have consulted the posting Guidelines on the Discussions front page
- [X] I have searched the Discussions forum and my question has not been asked before
- [X] I have searched the MOOSE website and the documentation does not answer my question
- [X] I have formatted my post following the posting guidelines (screenshots as a last resort, triple back quotes around pasted text)

### Question

Is there an existing implementation that can apply the neumann BC:

stress . normal = -d * du/dt?

Where d is a constant and du/dt is the time derivative of a variable (displacement)? I am trying to use this in conjunction with DynamicStressDivergenceTensors. I checked the existing ones and couldn't find one, wanted to know if there's an alternative way before I attempt implementing it myself.

**Comment by GiudGiud** [comment]
Hello

I went through the list and did not see one. You could search more in the dynamics examples

Guillaume

**Reply by richmondodufisan** [reply]
okay thank you
